src/motion/actuators/TransformAroundPointActuator.hx

```diff
@@ -78,13 +78,6 @@ class TransformAroundPointActuator<U> extends SimpleActuator<DisplayObject, U> {
 			tweenedOffsetY = value - originY;
 		} else {
 			super.setProperty(details, value);
-		}
-	}
-
-	private override function update(currentTime:Float) {
-		super.update(currentTime);
-
-		if ((active && !paused) || tweenPosition == 1) {
 			updatePosition();
```

Comment:
I think we should always call `updatePosition`, including cases with `x/y` property changes (although I'm not sure what those represent exactly).